spec/requests/api/v1/participant_declarations_spec.rb

```diff
@@ -14,6 +14,8 @@
   let(:participant_type)     { :ect }
   let(:milestone_start_date) { schedule.milestones.find_by(declaration_type: "started").start_date }
 
+  before { FeatureFlag.activate(:disable_npq) }
```

Comment:
I think it's worth removing the feature flag completely from those tests, after we fix the queries in v1/v2